ip update (#7)
GMLC-TDC/helics-conda: CONTRIBUTORS.md

# Contributors

> This file describes the contributors to the HELICS-conda project and the software used as part of this project It is part of the GMLC-TDC project and used in the HELICS project.  HELICS is a joint project between PNNL, LLNL, and NREL, with contributions from many other national Labs

Reviewer: ```suggestion
This file describes the contributors to the HELICS-conda project and the software used as part of this project. It is part of the GMLC-TDC project and used in the HELICS project.  HELICS is a joint project between PNNL, LLNL, and NREL, with contributions from many other national Labs
```

> If you would like to contribute to HELICS-cli, HELICS, or any of the GMLC-TDC related repositories see [CONTRIBUTING](CONTRIBUTING.md)

Reviewer: ```suggestion
If you would like to contribute to HELICS-conda, HELICS, or any of the GMLC-TDC related repositories see [CONTRIBUTING](CONTRIBUTING.md)
```

## Individual contributors
### National Renewable Energy Lab
 - Dheepak Krishnamurthy

### Pacific Northwest National Lab

### Lawrence Livermore National Lab
 - Philip Top
 - Ryan Mast

### Other

## Used Libraries or Code
### [HELICS](https://github.com/GMLC-TDC/HELICS)  

> Most of the original code for this library was pulled from use inside HELICS. Several examples were initially included in the HELICS repo but were pulled out when this repo was created  HELICS is released with a BSD-3-Clause license.

Reviewer: ```suggestion
The code used to build this package is pulled from the HELICS repository. HELICS is released with a BSD-3-Clause license.
```
